### ISSUE TITLE
carousel stops in between items when fast scrolling

### ISSUE THREAD

**Issue by aocenas** [DroidsOnRoids/SwiftCarousel#15]
![](http://g.recordit.co/qOiTyJH20S.gif)
As you can see in the last scroll, when I scroll fast enough it stops off center of any item. Gif is from simulator but the same thing happen on device. I thought it was side effect of PR: #12 but it seems this behavior was there before. Not really that annoying as I noticed it only now and as I said you have to scroll really fast.

**Comment by sunshinejr** [comment]
Yeah I've noticed it too, not really sure of a cause right on the spot tho. Will try to find it in a free time. Thanks for the issue! 🎉

**Comment by joelbanzatto** [comment]
Hello, I am facing same issue here. Do you have any updates?

**Comment by 1dividedby0** [comment]
I am also facing this issue. Any updates? @sunshinejr
